src/graphql/Bug.graphql

```diff
@@ -271,6 +271,29 @@ input BugSearch {
   whiteboards: [String]
 }
 
+type Comment {
+  # The globally unique ID for the comment.
```

Comment:
These all need to be camel-cased for consistency with the other API properties. There are mappings defined in https://github.com/mozilla-frontend-infra/bugzilla-graphql-gateway/blob/master/src/mappings.js that do this for the other APIs.